core/include/kernel/handle.h

```diff
@@ -22,6 +24,9 @@ struct handle_db {
  */
 void handle_db_destroy(struct handle_db *db, void (*ptr_destructor)(void *ptr));
 
+/* Checks if the associated pointers of all handles in the database are NULL. */
+bool handle_db_is_empty(struct handle_db *db);
```

Comment:
Please move this change and the one in `handle.c` into a separate commit.

Reply:
Done